### PR TITLE
Added PowersShell method for creating service principal

### PR DESCRIPTION
**Pull request by abdale**
Added PowersShell method for creatign service principal

### CODE REVIEW

**Review by marvinbuss — APPROVED** [review]
LGTM!

**Comment by marvinbuss** [comment]
@abdale Can you please submit the same in the other repos?

**Comment by abdale** [comment]
> @abdale Can you please submit the same in the other repos?

Yes, will do!

File: docs/DataManagementAnalytics-ServicePrincipal.md [hunk 1]
```diff
@@ -32,6 +32,20 @@ This will generate the following JSON output:
...
+
+```powershell
+#Ensure you are in the right subscription
+Set-AzContext -Subscription "{subscriptionName}"
+
+#Create service principal
+$sp = New-AzADServicePrincipal -DisplayName {name}
+$sp.PasswordCredentials.SecretText
+```
+The returned object contains the generated password. Make sure that you store this value somewhere secure to authenticate with the service principal.
```
Comment (marvinbuss) [inline docs/DataManagementAnalytics-ServicePrincipal.md:45]: Required for linting,
```suggestion
The returned object contains the generated password. Make sure that you store this value somewhere secure to authenticate with the service principal.
```

File: docs/DataManagementAnalytics-ServicePrincipal.md [hunk 2]
```diff
@@ -32,6 +32,20 @@ This will generate the following JSON output:
 
 > **Note:** Take note of the output. It will be required for the next steps.
 
+**Azure PowerShell:**
+
+```powershell
+#Ensure you are in the right subscription
```
Comment (marvinbuss) [inline docs/DataManagementAnalytics-ServicePrincipal.md:38]:
```suggestion
# Ensure you are in the right subscription
```

File: docs/DataManagementAnalytics-ServicePrincipal.md [hunk 3]
```diff
@@ -32,6 +32,20 @@ This will generate the following JSON output:
...
+Set-AzContext -Subscription "{subscriptionName}"
```
Comment (marvinbuss) [inline docs/DataManagementAnalytics-ServicePrincipal.md:39]: Both is allowed here.
```suggestion
Set-AzContext -Subscription "{SubscriptionId or SubscriptionName}"
```

File: docs/DataManagementAnalytics-ServicePrincipal.md [hunk 4]
```diff
@@ -32,6 +32,20 @@ This will generate the following JSON output:
...
+
+#Create service principal
```
Comment (marvinbuss) [inline docs/DataManagementAnalytics-ServicePrincipal.md:41]:
```suggestion
# Create service principal
```

**Review by marvinbuss — CHANGES_REQUESTED** [review]
Please see comments below.

**Review by marvinbuss — APPROVED** [review]
LGTM!

**Pull request merged**